src/main/java/dan200/computercraft/shared/peripheral/diskdrive/TileDiskDrive.java

```diff
@@ -524,9 +491,9 @@ private synchronized void ejectContents( boolean destroyed )
             }
 
             BlockPos pos = getPos();
-            double x = pos.getX() + 0.5 + (xOff * 0.5);
```

Comment:
Oh goodness, can we do this across the whole codebase please? Though also NBT cleanups and w/e.